distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Engine/Default.enso

```diff
@@ -827,19 +829,15 @@ type Match
    - id: The group identifier with which the error is associated.
 handle_error : Any -> (Text | Integer) -> Any
 handle_error error id = case error of
-    Polyglot_Error_Data err ->
+    Polyglot_Error.Polyglot_Error_Data err ->
         is_ioob = err.is_a IndexOutOfBoundsException
         is_iae = err.is_a IllegalArgumentException
         maps_to_no_such_group = is_ioob || is_iae
 
         if maps_to_no_such_group.not then err else
-            Regex.No_Such_Group_Error_Data id
+            No_Such_Group.Error id
     other -> other
 
-# TODO Dubious constructor export
-from project.Data.Text.Regex.Engine.Default.Option import all
-from project.Data.Text.Regex.Engine.Default.Option export all
-
 ## Options specific to the `Default` regular expression engine.
 type Option
```

Comment:
Do you think maybe it would be worth re-exporting the global `Regex_Options` in this `Option` type? So that we could mark that all options - both engine specific and global can be accessed through it?

(not sure though as it may cause confusion that the global ones are of different type, so there are pros and cons to this)

Reply:
The Regex design is not the final one - we are going to redo it in due course.
Hopefully the lower level engine will never be exposed to end users!